plugins/Morpheus/stylesheets/ui/_buttons.less

```diff
@@ -73,4 +73,11 @@ input[type="submit"].btn,
 
 .btn, .btn-large, .btn-small, .btn-flat {
     padding: 0 2rem;
+}
+
+.btn-close {
```

Comment:
This is a very global css class, but it contains very specific rules. The class is not reusable that way, or do you think every close button should be placed absolute? Might be better to move that where it is used and make the css selector more specific.